ports/glutin/window.rs

```diff
@@ -345,16 +345,16 @@ impl Window {
                     self.pending_key_event_char.set(Some(ch));
                 }
             }
-            Event::KeyboardInput(element_state, scan_code, Some(virtual_key_code)) => {
```

Comment:
Oh I see, looks like you're rolling in functionality for handling Some and None into the same match arm. Would it not be ok to just make a separate match arm instead specifically for the None case?